com.tiny-beast-dictionary.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.Arian94.Tiny-beast-dictionary
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: tiny-beast-dict
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+    --socket=pulseaudio
+  - --filesystem=xdg-documents
+modules:
+  - name: tiny-beast-dictionary
+    buildsystem: simple
+    build-commands:
+      - install -D src-tauri/tiny-beast-dictionary /app/bin/tiny-beast-dictionary
```

Comment:
this looks like a binary. How does that work with both x86_64 and aarch64.

Reply:
I only targeted x86_64. Does it have to support both?

Reply:
I have already published this app on snap. How does the "Software" App on Ubuntu recognize and merge flatpak and snap sources and show it as same software?

Reply:
Then you need to do that to ignore aarch64 https://github.com/flathub/flathub/wiki/App-Maintenance#flathubjson